modules/ROOT/pages/dataweave-language-guide.adoc

```diff
@@ -72,6 +72,12 @@ For example, `upper` is a built-in function defined in the String module. You ca
 
 DataWeave supports both _prefix_ notation (`function (param1, param2)`) and _infix_ notation (`param1 function param2`) for function calls. Note that you can use infix notation only with functions that accept only two parameters.
 
+=== Data Operators
+
+Data operators are symbols that enable you to perform operations on data to produce a result. DataWeave supports different types of operators including mathematical, relational, and logical operators. The operator notation varies depending on the operator. For example, mathematical and relational operators use infix notation: (2 `+` 2 `==` 4). Logical operators `and` and `or` also use infix notation, but logical operators `!` and `not` use prefix notation.
```

Comment:
```suggestion
Data operators are symbols that enable you to perform operations on data to produce a result. DataWeave supports different types of operators including mathematical (e.g. `+`, `-`), relational (e.g. `<`, `==`), and logical operators (e.g. `and`, `or`, `not`). There are other operators like flow control ones (like the `if else` operator), and an update operator to easily update specified fields of a data structure with new values.
```
Everything is correct but I'm not sure if the notation is important for this section of the language guide (it seems like a detail on operators). I would add a little bit of the flow control operator that is common in every language and the update operator (I think that is important for the user to know that they have those options when writing DW scripts).
And just enumerating some of those operators to give an idea to the user.